cborattr/src/cborattr.c

```diff
@@ -17,24 +17,29 @@
  * under the License.
  */
 
-//#include <syscfg/syscfg.h>
 #include "cborattr/cborattr.h"
 #include "tinycbor/cbor.h"
 #include "tinycbor/cbor_buf_reader.h"
 
+#ifdef __ZEPHYR__
+#include <zephyr.h>
+<<<<<<< Updated upstream
```

Comment:
I think you left this by accident.

Reply:
you are right

Reply:
I just checked my code locally, that is not here also the comment is marked as outdated.